utils/cloud_constants.py

```diff
@@ -1,55 +1,55 @@
-"""
+"""This file contains the constants for interaction with AWS/Minio."""
 
 import os
-import string
 
 # Please make sure you have your AWS envt variables setup
 AWS_S3_REGION = os.environ.get("AWS_S3_REGION", "ap-south-1")
 AWS_S3_ACCESS_KEY_ID = os.environ.get("AWS_ACCESS_KEY_ID", "")
 AWS_S3_SECRET_ACCESS_KEY = os.environ.get("AWS_SECRET_ACCESS_KEY", "")
-S3_BUCKET_NAME_MODEL = os.environ.get("AWS_BUCKET_NAME_MODEL", "avgupta-dev-gokube-triage")
-S3_BUCKET_NAME_INFERENCE = os.environ.get("AWS_BUCKET_NAME_INFERENCE", "avgupta-dev-gokube-triage")
+S3_BUCKET_NAME_MODEL = os.environ.get("AWS_BUCKET_NAME_MODEL",
+                                      "avgupta-dev-gokube-triage")
+S3_BUCKET_NAME_INFERENCE = os.environ.get("AWS_BUCKET_NAME_INFERENCE",
+                                          "avgupta-dev-gokube-triage")
 
 # Please set the following to point to your BQ auth credentials JSON
 BIGQUERY_CREDENTIALS_FILEPATH = os.environ.get(
     "BIGQUERY_CREDENTIALS_FILEPATH", "../../auth/bq_key.json"
 )
 
-GOKUBE_REPO_LIST = os.environ.get("GOKUBE_REPO_LIST", "/utils/data_assets/golang-repo-list.txt")
-KNATIVE_REPO_LIST = os.environ.get("KNATIVE_REPO_LIST", "/utils/data_assets/knative-repo-list.txt")
+GOKUBE_REPO_LIST = os.environ.get("GOKUBE_REPO_LIST",
+                                  "/utils/data_assets/golang-repo-list.txt")
+KNATIVE_REPO_LIST = os.environ.get("KNATIVE_REPO_LIST",
+                                   "/utils/data_assets/knative-repo-list.txt")
 KUBEVIRT_REPO_LIST = os.environ.get(
     "KUBEVIRT_REPO_LIST", "/utils/data_assets/kubevirt-repo-list.txt"
 )
 
 P1GRU_SEC_MODEL_TOKENIZER_PATH = os.environ.get(
     "P1GRU_SEC_MODEL_TOKENIZER_PATH",
-    "/model_assets/gokube-phase1-jun19/embeddings/security_tokenizer_word2idx_fulldata.pkl",
+    "/model_assets/gokube-phase1-jun19/embeddings/security_tokenizer_word2idx_fulldata.pkl",  # noqa
 )
 P1GRU_SEC_MODEL_WEIGHTS_PATH = os.environ.get(
     "P1GRU_SEC_MODEL_WEIGHTS_PATH",
-    "/model_assets/gokube-phase1-jun19/saved_models/security_model_train99-jun19_weights.h5",
+    "/model_assets/gokube-phase1-jun19/saved_models/security_model_train99-jun19_weights.h5",  # noqa
 )
 
 P1GRU_CVE_MODEL_TOKENIZER_PATH = os.environ.get(
     "P1GRU_CVE_MODEL_TOKENIZER_PATH",
-    "/model_assets/gokube-phase1-jun19/embeddings/cve_tokenizer_word2idx_fulldata.pkl",
+    "/model_assets/gokube-phase1-jun19/embeddings/cve_tokenizer_word2idx_fulldata.pkl",  # noqa
 )
 P1GRU_CVE_MODEL_WEIGHTS_PATH = os.environ.get(
     "P1GRU_CVE_MODEL_WEIGHTS_PATH",
-    "/model_assets/gokube-phase1-jun19/saved_models/cve_model_train99-jun19_weights.h5",
+    "/model_assets/gokube-phase1-jun19/saved_models/cve_model_train99-jun19_weights.h5",  # noqa
 )
 
 BASE_BERT_UNCASED_PATH = os.environ.get(
     "BASE_BERT_UNCASED_PATH",
-    "/model_assets/gokube-phase2/base_bert_tfhub_models/bert_uncased_L12_H768_A12",
+    "/model_assets/gokube-phase2/base_bert_tfhub_models/bert_uncased_L12_H768_A12",  # noqa
```

Comment:
All lines which has `# noqa ` does exceeds 80 char limit.

Reply:
@arajkumar in our projects we have a 100 character hard stop since f8a days. I believe all linters there should be configured this way because that standard is from 2017.

Reply:
As in it's the internal standard of the team, we follow 3 exception to True pep-8, forgot what the other two were. All my IDEs are setup that way.

Reply:
Can we use 120 line like we are using in osa-api ?
https://github.com/kubesecurity/osa-api-server/blob/master/setup.cfg

Reply:
fixed it!